src/mono/netcore/System.Private.CoreLib/src/System/Interop.Libraries.iOS.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class Libraries
+    {
+        internal const string SystemNative = "@executable_path/libSystem.Native.dylib";
+        internal const string GlobalizationNative = "@executable_path/libSystem.Globalization.Native.dylib";
+        internal const string NetSecurityNative = "@executable_path/libSystem.Net.Security.Native.dylib";
+        internal const string CryptoNative = "@executable_path/libSystem.Security.Cryptography.Native.OpenSsl.dylib";
+        internal const string CompressionNative = "@executable_path/libSystem.IO.Compression.Native.dylib";
+        internal const string IOPortsNative = "@executable_path/libSystem.IO.Ports.Native.dylib";
```

Comment:
Is `@executable_path` somehow recognized by the tool chain, e.g. the linker, and rewritten?  Or is that actually a directory special name recognized by iOS?

Reply:
`@executable_path` is expanded to the directory of the main executable by the dynamic loader (dyld) on Apple platforms: https://www.mikeash.com/pyblog/friday-qa-2009-11-06-linking-and-install-names.html (when `@executable_path` is encountered when loading dynamic libraries).

Reply:
Got it, thanks.